galite-core/src/main/kotlin/org/kopi/galite/visual/Module.kt

```diff
@@ -18,16 +18,49 @@
 
 package org.kopi.galite.visual
 
+import org.kopi.galite.base.Image
+import org.kopi.galite.db.DBContext
 import org.kopi.galite.l10n.LocalizationManager
+import org.kopi.galite.l10n.ModuleLocalizer
+import org.kopi.galite.util.base.InconsistencyException
 
+/**
+ * Represents an Module.
+ *
+ * @param id                  the ident of the module
+ * @param parent              the ident of the parent of the current module
+ * @param shortname           the shortname
+ * @param source              the source
+ * @param objectName          the name of the object which are linked with the module
+ * @param access              the access of the current user
+ * @param priority            the priority
+ * @param icon                the mnemonic
+ */
 class Module(val id: Int,
              val parent: Int,
-             shortname: String,
-             source: String,
-             val objectName: String?,
-             var accessibility: Int,
-             priority: Int,
-             icon: String?) : Comparable<Module?> {
+             private val shortname: String,
+             private val source: String,
+             objectName: String?,
+             var access: Int,
```

Comment:
change access to accessibility
`@param accessibility the access of the current user to the module`